.github/workflows/build-deploy.yml

```diff
@@ -26,29 +46,25 @@ jobs:
       ORG_GRADLE_PROJECT_signingKey: ${{ secrets.CODE_SIGNING_PRIV_KEY }}
       SONATYPE_PW: ${{ secrets.SONATYPE_PW }}
       SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
do we need these two SONATYPE_PW and SONATYPE_USERNAME env variables to build?

Reply:
Yes, I had moved them by mistake to the sonar-check job and it failed. They are required for publishing the artifacts.